CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 ### Added
 - Documentation of sink (`SINK.md`).
 
+### Fixed
+- Import failed for transactions with `tour_id` exceeding the INTEGER
+  range.
+
```

Comment:
This commit doesn't fix the `tour_id` column only this. This commit widens space for several integer columns.